test/proto2pb/test_all_types.proto

```diff
@@ -112,7 +112,7 @@ message NestedTestAllTypes {
 // with fully qualified names.
 message ExampleType {
   optional string name = 1;
-
+  optional int64 in = 2;
```

Comment:
Can you make sure to update the canonical test protos with the same field name in a PR on cel-spec?

Reply:
Will do.